### PR TITLE
Fix for dropbox/nsot#260

### PR DESCRIPTION
**Pull request by rmhasan**
Added all subcommands to query new endpoints in
Interface view

New interface list subcommands include
-parent
-root
-ancestors
-children
-descendants
-siblings

### CODE REVIEW

File: pynsot/commands/cmd_interfaces.py [hunk 1]
```diff
@@ -343,6 +343,58 @@ def addresses(ctx, *args, **kwargs):
     )
 
 
+@list.command()
+@click.pass_context
+def parent(ctx, *args, **kwargs):
+    """Get parent of an Interface"""
```
Comment (jathanism) [inline pynsot/commands/cmd_interfaces.py]: Just me nitpicking again... Can you please mirror the similar comments from the Network commands of the same names? I'm just coming from a CLI user-experience perspective!

**Comment by rmhasan** [comment]
Hi @jathanism, I made some changes to the doc strings of the new `Interface` tree traversal methods.

**Comment by nickpegg** [comment]
I went ahead and opened a PR for that IPython fix: #135 

@rmhasan once #135 is merged, you can rebase against `develop` and tests should start passing again.

Besides that, this PR LGTM!

**Comment by jathanism** [comment]
Once we get it rebased and the tests pass we're good to go!

**Comment by rmhasan** [comment]
Thanks @nickpegg, I rebase this PR.

**Comment by nickpegg** [comment]
Thanks again for adding this feature @rmhasan! I've cut v1.1.4 for pynsot and it's up on PyPI now

**Pull request merged**